scripts/gen_syscall_header.py

```diff
@@ -88,7 +88,7 @@ def gen_make_syscall(ret, argc, tabcount):
 def gen_call_impl(ret, argc):
     if (ret != Retval.VOID):
         sys.stdout.write("return ")
-    sys.stdout.write("_impl_##name(")
+    sys.stdout.write("z_sys_impl_##name(")
```

Comment:
why not z_impl_?

I don't like the additional _sys_ here, the names are long enough already

Reply:
z_sys_impl_ was taken from #9882, we can change it so something more reasonable.

Reply:
Updated.